api/src/main/java/org/openmrs/User.java

```diff
@@ -35,30 +59,59 @@
  * key-value pairs for either quick info or display specific info that needs to be persisted (like
  * locale preferences, search options, etc)
  */
+@Entity
+@Table(name = "users")
+@Cacheable
+@Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
+@SecondaryTable(name = "users_unused_fields")
```

Comment:
Can we put some code comments explaining why we are introducing the users_unused_fields table?

Reply:
Done